README.md

```diff
@@ -22,12 +22,66 @@ The master branch works with **PyTorch 1.3+**.
   <img src="demo/demo.gif" width="600px"/>
 </div>
 
+
+### Major Features
+
+- **Modular design**
+
```

Comment:
move this item to the bottom since it is quite standard now

Reply:
OK.